src/renderer/components/_molecules/rung/body.tsx

```diff
@@ -161,7 +161,23 @@ export const RungBody = ({ rung }: RungBodyProps) => {
       const [type, library, pouName] = blockType.split('/')
       if (type === 'system')
         pouLib = libraries.system.find((lib) => lib.name === library)?.pous.find((p) => p.name === pouName)
-
+      if (type === 'user') {
```

Comment:
Avoid using word abbreviations.

```
pouLib = pouLibrary
lib = Library
```

If the context requires, assign names to be more descriptive.

```
const [ type, library, pouName ] = blockType.split('/')
const [ blockLibraryType, blockLibrary, pouName ] = blockType.split('/')
```